# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/bug1681664_helper.js
# Commit: 490dbdb4e499
# Full Hash: 490dbdb4e4996b91ab7aa8b4dd70de4029602c73
# Author: yulia <ystartsev@mozilla.com>
# Date: 2021-01-12 04:33:40
# Regressor Bug: 1681664
# File Overlap Count: 2
# Description:
#   Bug 1681664 - Allow Top-level await modules which fail to evaluate due to user action to fail without crashing. r=jonco,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99809
# ==============================================================================

diff -r 53327022537a -r 490dbdb4e499 js/xpconnect/tests/mochitest/bug1681664_helper.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/xpconnect/tests/mochitest/bug1681664_helper.js	Mon Jan 11 17:08:59 2021 +0000
@@ -0,0 +1,1 @@
+while (true) {};